ingestion-data/staging/collections/nldas2.json

```diff
@@ -0,0 +1,116 @@
+{
+    "id": "nldas2",
+    "type": "Collection",
+    "title": "NLDAS2 precipitation",
+    "links": [],
+    "description": "NLDAS-2 provides a fine-scale (1 km) meteorological forcing (precipitation) in both retrospective and near real-time over North and Central America, including Alaska, Hawaii, and Puerto Rico, by leveraging high-quality gauge, satellite, and model datasets through advanced data assimilation methods. Read more: https://ldas.gsfc.nasa.gov/nldas/v2",
+    "extent": {
+        "spatial": {
+            "bbox": [
+                [
+                    -41.2,
+                    -82.6,
+                    9.8,
+                    -59
+                ]
+            ]
+        },
+        "temporal": {
+            "interval": [
+                [
+                    "2001-01-01T00:00:00Z",
+                    "2023-06-01T00:00:00Z"
+                ]
+            ]
+        }
+    },
+    "license": "MIT",
+    "stac_extensions": [
+        "https://stac-extensions.github.io/render/v1.0.0/schema.json",
+        "https://stac-extensions.github.io/item-assets/v1.0.0/schema.json"
+    ],
+    "item_assets": {
+        "cog_default": {
+            "type": "image/tiff; application=geotiff; profile=cloud-optimized",
+            "roles": [
+                "data",
+                "layer"
+            ],
+            "title": "Default COG Layer",
+            "description": "Cloud optimized default layer to display on map"
+        }
+    },
+    "dashboard:is_periodic": false,
+    "dashboard:time_density": "month",
+    "stac_version": "1.0.0",
+    "renders": {
+        "dashboard": {
+            "colormap": {
```

Comment:
```suggestion
            "color_map": {
```

Reply:
Are you sure?
![image](https://github.com/user-attachments/assets/1a3ba155-21f9-4e80-ace7-09ffdb26c929)

Reply:
Sorry, I was wrong. Choices are `colormap_name` for a predefined option or `colormap` for a custom colormap (like what you already have).

I thought it might be why the preview doesn't work for the items in this collection but I just realized we haven't updated renders for the upgrades to titiler-pgstac. New issue to do that: https://github.com/NASA-IMPACT/veda-backend/issues/406

Reply:
@anayeaye that was a good catch! Forgot about renders using the raster api!